python/ray/autoscaler/kuberay/ray-cluster.complete.yaml

```diff
@@ -111,6 +115,16 @@ spec:
               name: autoscaler-config
             - mountPath: /tmp/ray
               name: ray-logs
+          env:
+          - name: RAY_CLUSTER_NAMESPACE
+            valueFrom:
+              fieldRef:
+                fieldPath: metadata.namespace
+          - name: RAY_CLUSTER_NAME
+            # This value must match the metadata.name of the RayCluster CR.
+            # The autoscaler uses this env variable to determine which Ray CR to interact with.
+            # TODO: Match with CR name automatically via operator, Helm, and/or Kustomize.
+            value: raycluster-complete
```

Comment:
why not to use fieldRef as namespace?

Reply:
`fieldRef` supports certain fields of a pod
I don't know of a good way to reference the name of the CR.

The fieldRef used for the namespace is a reference to the namespace of the Ray head pod, which happens to be the same as the name of the RayCluster CR.